### PR TITLE
Update Security Arabic Translations

### PR DESCRIPTION
**Pull request by zairigimad**
| Q             | A
| ------------- | ---
| Branch?       | 5.x for features / 3.4, 4.4 
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | Fix #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Related to https://github.com/symfony/symfony/issues/38723

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Resources/translations/security.ar.xlf [hunk 1]
```diff
@@ -66,6 +66,15 @@
                 <source>Account is locked.</source>
                 <target>الحساب مغلق.</target>
             </trans-unit>
+            <trans-unit id="17">
+                <source>Too many failed login attempts, please try again later.</source>
+                <target>عدد كبير جدا من محاولات الدخول الفاشلة، يرجى المحاولة مرة أخرى في وقت لاحق.
+</target>
```
Comment (fabpot) [inline src/Symfony/Component/Security/Core/Resources/translations/security.ar.xlf]: Extra newline apparently.

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Security/Core/Resources/translations/security.ar.xlf (hunk 1 above)
Comment (zairigimad) [reply src/Symfony/Component/Security/Core/Resources/translations/security.ar.xlf]: you are right, fixed in the last commit :)

**Review by Nyholm — APPROVED** [review]
Thank you

**Review by derrabus — APPROVED** [review]

**Comment by derrabus** [comment]
Thank you @zairigimad.

**Pull request merged**